gradio/components.py

```diff
@@ -4181,10 +4218,15 @@ def update(
             color_legend_title: The title given to the color legend. By default, uses the value of color parameter.
             size_legend_title: The title given to the size legend. By default, uses the value of the size parameter.
             shape_legend_title: The title given to the shape legend. By default, uses the value of the shape parameter.
+            color_legend_position: The position of the color legend. If 'none', omit the legend. For other valid position values see: https://vega.github.io/vega/docs/legends/#orientation.
+            size_legend_position: The position of the size legend. If 'none', omit the legend. For other valid position values see: https://vega.github.io/vega/docs/legends/#orientation.
+            shape_legend_position: The position of the shape legend. If 'none', omit the legend. For other valid position values see: https://vega.github.io/vega/docs/legends/#orientation.
```

Comment:
Just to clarify, what does "If 'none', omit the legend" mean? if *any* of these `None`, the legend is omitted?